libr/anal/vtable.c

```diff
@@ -87,9 +87,10 @@ static bool vtable_section_can_contain_vtables(RVTableContext *context, RBinSect
 	if (section->is_segment) {
 		return false;
 	}
-	return !strcmp (section->name, ".rodata") ||
-		   !strcmp (section->name, ".rdata") ||
-		   !strcmp (section->name, ".data.rel.ro");
+	bool r = !strcmp (section->name, ".rodata") ||
+		!strcmp (section->name, ".rdata") ||
+		!strcmp (section->name, ".data.rel.ro") ||
+		r_str_endswith (section->name, "__const");
```

Comment:
missing return r; :PPP

Reply:
lol maybe I should actually compile my code before committing.

Reply:
:D